app/src/main/java/to/dev/dev_android/view/main/view/MainActivity.kt

```diff
@@ -23,10 +23,15 @@ class MainActivity : BaseActivity<ActivityMainBinding>(), CustomWebChromeClient.
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setWebViewSettings()
-        navigateToHome()
+        savedInstanceState?.let { restoreState(it) }?: navigateToHome()
```

Comment:
Nitpicking
Edit spacing around `?:` for consistency. Making it like:
`savedInstanceState?.let { restoreState(it) } ?: navigateToHome()`

Reply:
Thanks for the heads up, on it